src/pages/components/checkbox.mdx

```diff
@@ -1,15 +1,184 @@
 ---
 title: Checkbox
 description: >
-  Description will be here
+  A Checkbox lets users select and unselect options from a list.
+package: '@zendeskgarden/react-forms'
+components:
+  - forms/src/elements/Checkbox.tsx
+  - forms/src/elements/common/Field.tsx
+  - forms/src/elements/common/Hint.tsx
+  - forms/src/elements/common/Label.tsx
+  - forms/src/elements/common/Message.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To let users compare options from a list and select all, any, or none of those items.</li>
```

Comment:
Line 17 shouldn't have closing punctuation:

`<li>To let users compare options from a list and select all, any, or none of those items</li>`

Reply:
Updated, thanks V!